docker-app/qfieldcloud/core/serializers.py

```diff
@@ -535,3 +546,44 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    organization = serializers.StringRelatedField(source="team_organization")
+
+    class Meta:
+        model = Team
+        fields = ("username", "organization")
+
+
+class TeamMemberSerializer(serializers.ModelSerializer):
+    member = serializers.CharField()
+
+    def to_internal_value(self, data: Dict[str, Any]) -> Dict[str, Any]:
+        validated_data = super().to_internal_value(data)
+        email_or_username = data.get("member")
```

Comment:
```suggestion
        email_or_username: str = data["member"]
```

We can safely assume the value will always be present in the dict, as the Serializer will automatically check for it. I might be wrong, to be verified.

Reply:
I will check it!

Reply:
Are you going to check it :P